src/main/markdown/makinggwtbetter.md

```diff
@@ -505,26 +482,11 @@ parameters), the name should be capitalized and have suffix "T". In a
 nutshell, prefer `<T>` or `<K, V>`, and devolve to `<KeyT, ValueT>` if
 necessary.
 
-<table style="margin-left: 2em">
-  <tr>
-    <th style="width: 16em">Good</th>
-    <th>Bad</th>
-    <th>Tolerable</th>
-  </tr>
-  <tr>
-    <td>`Foo<T>`</td>
-    <td>`Foo<FooClientType>`</td>
-  </tr>
-  <tr>
-    <td>`Bar<K, V>`</td>
-    <td>`Bar<KeyType, ValueType>`</td>
-  </tr>
-  <tr>
-    <td>`Baz<V, E, X>`</td>
-    <td>`Baz<EventType, ErrorType, ExpressionType>`</td>
-    <td>`Baz<EventT, ErrorT, ExpressionT>`</td>
-  </tr>
-</table>
+|Good          |Bad                                        |Tolerable                         |
+|--------------|-------------------------------------------|
+|`Foo<T>`      |`Foo<FooClientType>`                       |
+|`Bar<K, V>`   |`Bar<KeyType, ValueType>`                  |
```

Comment:
It depends on whether we want an empty third column. Currently there's none. I removed them to match the current look.